### ISSUE TITLE
Implement trending article feature

### ISSUE THREAD

**Issue by psiemens** [ubyssey/ubyssey-dispatch-theme#211]
For this feature, we want to identify trending articles and feature them on the homepage.

An article is considered to be "trending" if it meets the following criteria (exact numbers subject to change):

- It was published in the last 6 hours
- It has more than 1000 views

The homepage should display the trending article with the most views in the top left corner of the page. There might not always be a trending article, but that's okay.

![screen shot 2017-03-07 at 7 47 36 pm](https://cloud.githubusercontent.com/assets/2547035/23689379/f82f8930-036e-11e7-9e13-5351cbfa17f4.png)

Here's the code for the trending icon:

```css
.o-trending-badge {
    color: white;
    width: 22px;
    height: 22px;
    border-radius: 22px;
    background: #fd3333;
    text-align: center;
    line-height: 22px;
    letter-spacing: 0;
    margin-right: 2px;
    font-size: 13px;
}
```

```html
<i class="o-trending-badge fa fa-bolt" aria-hidden="true"></i>
```

I'll let you figure out how to perform a query to find trending articles. This is a good place to start: https://docs.djangoproject.com/en/1.10/topics/db/queries/#retrieving-specific-objects-with-filters

**Comment by aprilzhong** [comment]
I'd like to work on this!

**Comment by psiemens** [comment]
Tentative deadline: **Friday, November 10th**

**Comment by jumbosushi** [comment]
Closed by #425
